oak_channel/SPEC.md

```diff
@@ -44,91 +43,47 @@ The service MUST NOT send request messages.
 ##### Handling an invocation
 
 The service MUST respond to each request message with a single response message.
-It SHOULD respond to request messages in first-in-first-out order, but MAY
-prioritize some request messages. The service MAY send a response message before
-the request message has been received in its entirety. Sending a response
-message concludes the invocation.
+The service MAY send a response message before the request message has been
+received in its entirety, but MUST NOT send start sending a response before any
```

Comment:
IIUC "it its entirety" implies that messages be only partially sent/received, i.e., not atomic.

If so, what does: "Sending a response message" mean excatly below? The point where the service starts sending a message or the entire message is sent?

Reply:
The point when the complete message has been transmitted.